src/JSON.jl

```diff
@@ -1,3 +1,5 @@
+VERSION >= v"0.4.0-dev+6521" && __precompile__()
```

Comment:
Not a bad suggestion.

But, I just submitted a whole slew of these, and I'm not gonna redo them without stronger reason than that :smile:.